nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -99,14 +101,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 -input $(CsWinRTInteropMetadata)
 -include WinRT.Interop
       </CsWinRTIncludeWinRTInterop>
+      <CsWinRTEmbeddedParam Condition="'$(CsWinRTEmbedded)' == 'true'">-embedded</CsWinRTEmbeddedParam>
+      <CsWinRTExeTFM Condition="'$(TargetFramework)' == 'netcoreapp3.1'">netstandard2.0</CsWinRTExeTFM>
```

Comment:
If an app using embedded targets netcoreapp3.1, then cswinrt.exe complains.

Reply:
The other special case would be `net48`. We don't support it today. Should determine if we want to support embedded projections for .NET Framework.

Reply:
Embedded should work with any `netstandard2.0` compatible framework right? Looking at these docs: https://docs.microsoft.com/dotnet/standard/net-standard
.NET Standard 2.0 supports earlier versions down to `net46`. and `netcoreapp2.0` @AdamBraden do you have thoughtson what specific TFM values (for .NET Core and .NET Framework) embedded should support?

Reply:
We should only support TFM=netstandard2.0 and NET5/6+. I'm not aware of customer scenario/requirement to explicitly support something like net48 or netcoreapp2.1

Reply:
We get support for net47/48 for free with a tweak to this line. I tested this in a new sample app in this PR, showing that an SDK style app can target net48 and embed a WinRT projection into the .dll.